### PR TITLE
Add a draft for the Q4 2023 update

### PR DESCRIPTION
**Pull request by dnadales**

### CODE REVIEW

File: quarterly/2023-Q4-consensus.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+### Support vendors to deliver contracts
+
+- TODO: highlight collaboration between Tweag, Nick and Alex
+- TODO: highlight collaboration between Javier and Joris.
+
+### Operation serenity Q4 2023
+
+- ...
+- We added a test to check Consensus emits valid CBOR, which prevents the generation of invalid binary encoding.
```
Comment (dnadales) [inline quarterly/2023-Q4-consensus.md:54]:
```suggestion
- We added tests to check Consensus emits valid CBOR, which prevents the generation of invalid binary encoding.
```

File: quarterly/2023-Q4-consensus.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+### Assist with repo transfer to Intersect
+
+### Support vendors to deliver contracts
+
+- TODO: highlight collaboration between Tweag, Nick and Alex
+- TODO: highlight collaboration between Javier and Joris.
+
+### Operation serenity Q4 2023
+
+- ...
```
Comment (dnadales) [inline quarterly/2023-Q4-consensus.md:53]: Mention new team member @RenateEilers :tada:

File: quarterly/2023-Q4-consensus.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+### Assist with P2P IOG relay network shut down
+
+### Assist with repo transfer to Intersect
+
+### Support vendors to deliver contracts
+
+- TODO: highlight collaboration between Tweag, Nick and Alex
+- TODO: highlight collaboration between Javier and Joris.
```
Comment (dnadales) [inline quarterly/2023-Q4-consensus.md:49]: Let's mention the presentations and milestones reached regarding LSM.

**Review by jorisdral — APPROVED** [review]
:+1: on the UTxO-HD side

**Pull request merged**